code/modules/mob/inventory.dm

```diff
@@ -279,20 +279,25 @@
 	return 0
 
 
-/mob/proc/drop_from_inventory(var/obj/item/W) //I'm fairly sure the entirety of this proc is redundant and can be replaced by just u_equip(W,1)
+/mob/proc/drop_from_inventory(var/obj/item/W, var/delete = FALSE) //I'm fairly sure the entirety of this proc is redundant and can be replaced by just u_equip(W,1)
```

Comment:
why?

Reply:
In addition to being stupid, this change is, as far as I can tell, unused

Reply:
misc_gamemode_procs.dm#L276 calls `H.drop_all(TRUE)`
`drop_all()` calls `drop_from_inventory` for every item in the inventory
I guess @MadmanMartian forgot a line. I'll fix it.

Reply:
equip_weeaboo proc uses it, through drop_all

Reply:
Did it pass the var before I added that line?

Reply:
I'd just make it a separate proc that just `qdel`s everything equipped, `item/Destroy()` already calls `drop_from_inventory` if necessary

Reply:
Alright, I'll make a seperate proc.